### PR TITLE
Unqualified Dublin Core RDF.js -- fix MOST eslint warnings

### PR DESCRIPTION
**Pull request by retorquere**
I don't know where dcterms is supposed to come from

### CODE REVIEW

**Comment by retorquere** [comment]
updated for @zotero/eslint-config@1.0.2

File: Unqualified Dublin Core RDF.js [hunk 1]
```diff
@@ -82,42 +108,44 @@ function doExport() {
...
 		// archiveLocation
 		if (item.archiveLocation) {
-			Zotero.RDF.addStatement(resource, dc+"coverage", item.archiveLocation, true);
+			Zotero.RDF.addStatement(resource, dc + "coverage", item.archiveLocation, true);
 		}
 		
 		// medium
 		if (item.medium) {
-			Zotero.RDF.addStatement(resource, dcterms+"medium", item.medium, true);
+			Zotero.RDF.addStatement(resource, dc + "medium", item.medium, true);
```
Comment (zuphilip) [inline Unqualified Dublin Core RDF.js]: There is no `medium` property in DC elements http://purl.org/dc/elements/1.1/ but in DC terms http://purl.org/dc/terms . However, this translator is named "Unqualified Dublin Core RDF" which would be just DC elements. Hm... we have to think this further.

File: Unqualified Dublin Core RDF.js [hunk 2]
```diff
@@ -11,25 +11,50 @@
...
 	},
 	"inRepository": true,
-	"lastUpdated": "2018-12-17 03:16:21"
+	"lastUpdated": "2019-06-11 13:33:25"
 }
 
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018-2019 Simon Kornblith
```
Comment (zuphilip) [inline Unqualified Dublin Core RDF.js]: Did you speak with him? Any reason for the time spans here where Simon was not actively working on this?

File: Unqualified Dublin Core RDF.js (hunk 1 above)
Comment (zuphilip) [reply Unqualified Dublin Core RDF.js]: Actually, I think we can use `dc:format` here, which stands for "The file format, physical medium, or dimensions of the resource." http://www.dublincore.org/specifications/dublin-core/dcmi-terms/#elements-format . But this then should go into a separate commit.

File: Unqualified Dublin Core RDF.js (hunk 2 above)
Comment (retorquere) [reply Unqualified Dublin Core RDF.js]: I haven't, no, but I can.

Comment (zuphilip) [reply Unqualified Dublin Core RDF.js]: This might not be needed, cf. https://github.com/zotero/translators/issues/951#issuecomment-145275715

Comment (zuphilip) [reply Unqualified Dublin Core RDF.js]: Then I simply suggest to change the date to the original creation date to indicate this is a older translator. Maybe we also should put the relicensing commit into a separate commit.

File: Unqualified Dublin Core RDF.js (hunk 1 above)
Comment (retorquere) [reply Unqualified Dublin Core RDF.js]: OK, so what should I do in the interim?

File: Unqualified Dublin Core RDF.js (hunk 2 above)
Comment (retorquere) [reply Unqualified Dublin Core RDF.js]: Which date? The date in the license block? So just have 2018 in the license block?

Comment (zuphilip) [reply Unqualified Dublin Core RDF.js]: 2008 should be okay

Comment (retorquere) [reply Unqualified Dublin Core RDF.js]: done

File: Unqualified Dublin Core RDF.js (hunk 1 above)
Comment (zuphilip) [reply Unqualified Dublin Core RDF.js]: I will handle that...

**Comment by zuphilip** [comment]
Thank you!

**Pull request merged**